vendor/JSON-Schema-Test-Suite/.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Ping the entire test suite team by default.
+*       @json-schema-org/test-suite-team
```

Comment:
GGitHub linter shows an error here:

```
Unknown owner on line 2: make sure the team @json-schema-org/test-suite-team exists, is publicly visible, and has write access to the repository

*       @json-schema-org/test-suite-team
```

Reply:
These files are part of the vendored source from `JSON-Schema-Test-Suite`. Since it's not a submodule anymore they show up as if they were regular files in this repository. It'd be nice to not have to do that, but the flake build doesn't fetch git submodule by default.